src/drakvuf.h

```diff
@@ -118,6 +118,7 @@
 #include <plugins/plugins.h>
 #include <libinjector/libinjector.h>
 
+#ifndef PRINT_DEBUG
```

Comment:
I don't follow the logic behind this extra `ifndef`, care to explain?

Reply:
I had plenty compile errors while trying to use `PRINT_DEBUG` macro, so I've added an `#ifndef` (as it doesn't hurt, but solves a potential problem).

Reply:
Well, it doesn't hurt, but you might be papering over some other issue with this. What were the compile errors? There shouldn't be any.

Reply:
Redefinition of a macro

Reply:
Gotcha.. sigh